Multithreading Brute force HW/Sources/ViewController/ViewController.swift

```diff
@@ -0,0 +1,344 @@
+//
+//  ViewController.swift
+//  Multithreading Brute force HW
+//
+//  Created by Serhii  on 12/10/2022.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+
+    var isBlack: Bool = false {
+        didSet {
+            if isBlack {
+                view.backgroundColor = .black
+                statusLable.textColor = .red
+                textLable.textColor = .red
+                activityIndicator.color = .red
+                hackerImage.tintColor = .white
+
+            } else {
+                view.backgroundColor = .systemGray2
+                statusLable.textColor = .black
+                textLable.textColor = .black
+                activityIndicator.color = .black
+                hackerImage.tintColor = .black
+            }
+        }
+    }
+
+    var isStarted: Bool = false {
+        didSet {
+            if isStarted {
+                statusLable.text = "Взлом пароля 😈😈😈"
+                startButton.backgroundColor = .systemRed
+                startButton.setTitle("Stop", for: .normal)
+                randomStack.isHidden = true
+            } else {
+                statusLable.text = ""
+                startButton.backgroundColor = .systemGreen
+                startButton.setTitle("Start", for: .normal)
+                randomStack.isHidden = false
+            }
+        }
+    }
+
+    // MARK: - Outlets
+
+    private lazy var hackerImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "hacker")
+        return imageView
+    }()
+
+    private lazy var textLable: UILabel = {
+        let lable = UILabel()
+        lable.font = UIFont.systemFont(ofSize: 20, weight: .medium)
+        return lable
+    }()
+
+    private lazy var statusLable: UILabel = {
+        let lable = UILabel()
+        lable.textColor = .red
+        lable.font = UIFont.systemFont(ofSize: 20, weight: .medium)
+        return lable
+    }()
+
+    private lazy var passwordTextField: UITextField = {
+        let textField = UITextField()
+        textField.attributedPlaceholder = NSAttributedString(
+            string: "Введие пароль",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.lightGray])
```

Comment:
Зачем так сложно, если можно просто
```suggestion
        textField.placeholder = "Введите пароль"
```